build/media_source/plg_system_shortcut/keyselect.es6.js

```diff
@@ -0,0 +1,81 @@
+class JoomlaShortcutModal {
+	constructor() {
+		
+		if (!Joomla) {
+			throw new Error('Joomla API is not properly initialised');
+		}
+		// Bindings
+    	this.setAttributes = this.setAttributes.bind(this);
+		this.buildKeySelectModal = this.buildKeySelectModal.bind(this);
+
+		document.addEventListener("DOMContentLoaded",this.buildKeySelectModal);
```

Comment:
use single quote `document.addEventListener('DOMContentLoaded',this.buildKeySelectModal);`

And every where